src/PartielsPy/export_configs/export_config_image.py

```diff
@@ -0,0 +1,111 @@
+"""A class for export image configuration, used for jpeg and png formats"""
+
+from enum import Enum
+
+from ._export_config import ExportConfig
+
+
+class ExportConfigImage(ExportConfig):
+    class FormatEnum(Enum):
+        """Enum for the image format"""
+
+        JPEG = "jpeg"
+        PNG = "png"
+
+    def __init__(
+        self,
+        format: str = FormatEnum.JPEG.value,
+        width: int = 1280,
+        height: int = 720,
+        groups: bool = False,
+    ):
+        """Initialize the export configuration for the image.
+
+        This class is used to configure the export of the image.
+        Images format is jpeg or png.
```

Comment:
Perhaps the default values could be specified in the Args section, for ex. `width (int): the width of the image in pixels (default: 1280 pixels)`